CorsixTH/Lua/app.lua

```diff
@@ -1534,7 +1534,7 @@ function App:restart()
       self.ui:addWindow(UIInformation(self.ui, {_S.information.cannot_restart}))
       return
     end
-    local status, err = pcall(self.loadLevel, self, level, difficulty, name, file, intro)
+    local status, err = pcall(self.loadLevel, self, level, difficulty, name, file, intro, nil, true)
```

Comment:
Adding more parameters gets tricky at some point, there is a thing named "too many parameters", and to me, this one is one of them.
So the question is whether you can avoid adding parameters, and if possible, even reduce the number of parameters.

I can see at least 2 options.
1. Playing the movie seems to not affect other parts of that function. Thus, it should be possible to pull that play code out of the function. Then add it again as a separate call after a call to the above function. At this spot you can simply leave it out completely, although a comment with a reason why it is not played here is likely useful to avoid someone else trying to be consistent and add it again.
2. Break the `loadLevel` function into pieces, and call each piece separately.

Both alternatives somewhat prefer this being a separate patch, as the amount of changes grows.